i18n/en.toml

```
@@ -85,3 +85,15 @@ other = "Was this page helpful?"
 other = "Yes"
 [feedback_answer_no]
 other = "No"
+
+# Sign language section
+[sign_language_header]
+other = "In sign language"
+
+[sign_language_info]
+other = """
+While each country has its own sign language, 
+we'd like to standardize signs as much as possible for new cloud native terms.
+When a cloud native term is composed of words that already exist in English, 
+we'll use American Sign Language (ASL).
```

Here's a copyedited message, with links to appropriate terms:

```suggestion
**Note**:
While each country has its own sign language we, the
[Deaf/Hoh Working Group](https://contribute.cncf.io/accessibility/deaf-and-hard-of-hearing/),
aim to standardize signs for new cloud-native terms. When a cloud-native term
consists of words that already exist in English, we will use
[American Sign Language](https://www.nidcd.nih.gov/health/american-sign-language) (ASL).
```